README.md

```diff
@@ -327,3 +326,46 @@ const apos = require('apostrophe')({
   }
 });
 ```
+
+### 5 Ignoring phrases that come from Apostrophe's admin UI
+
+For some projects, you may not be interested in translating the phrases that make up Apostrophe's admin UI.
+
+The first step is to tell ApostropheCMS to "namespace" those phrases:
+
+```js
+modules: {
+  'apostrophe-i18n': {
+    namespaces: true
```

Comment:
With this configuration, I have this behavior:

![image](https://user-images.githubusercontent.com/11479686/78334341-e1f42800-758b-11ea-8abb-3090188b5797.png)

Wordings from the interface have this "> tags" value everywhere